src/build.py

```diff
@@ -694,6 +693,23 @@ def Which(name):
     return ''
 
 
+def CopyLLVMTools(build_dir, install_subdir=''):
```

Comment:
In autoconf terminology this is called the PREFIX, and the its appended to the DESTDIR.   So you might configure with PREFIX=/usr/local and then install with DESTDIR=/tmp/location, and you'd end up with all your files in /tmp/location/usr/local.